src/components/playback/playmethodhelper.js

```diff
@@ -3,7 +3,9 @@ export function getDisplayPlayMethod(session) {
         return null;
     }
 
-    if (session.TranscodingInfo && session.TranscodingInfo.IsVideoDirect) {
+    if (session.TranscodingInfo && session.TranscodingInfo.IsVideoDirect && session.TranscodingInfo.IsAudioDirect) {
+        return 'Remux';
+    } else if (session.TranscodingInfo && session.TranscodingInfo.IsVideoDirect) {
         return 'DirectStream';
```

Comment:
@MrTimscampi don't we want to remove this term entirely? Might as well do it now if that's the case.

Reply:
It depends if we want to separate a transcode with the video from one without.

Imo, having more terms make no sense, so we may as well remove it, yeah. It's either a transcode or a remux, there's nothing in-between (and we have the playback data for more info as to what is happening)

Reply:
Transcoding audio on the fly isn't a thing comparing to the video re-encoding. Thus separating (vcopy+a copy) and (vcopy+acodec) will just confuse users.

Reply:
I want to keep it because of ffmpeg behavior. take for example our keyframe bugs.

jellyfin/jellyfin#2547 / jellyfin/jellyfin-web#879 / https://github.com/jellyfin/jellyfin-android/issues/301 / jellyfin/jellyfin-expo#54 Audio/Video playback position wrong after changing playback position due to non-keyframe HLS use

this bug impacts Remux and DirectStream Only.

Reply:
To update all:

Direct Play - Lossless Process
Remux - Lossless process, invokes FFMpeg to HLS stream. Suffers from keyframe issues
DirectStream - Converts Audio, Can be process intensive for some low powered machines when converting something such as 7.1 DTS audio. Suffers from keyframe issues.
Transcode - Video Transcoding. Audio doesnt matter.

I'd like to keep all 4 terms. Each scenario has its own issues and uniqueness. I'm down for renaming DirectStream but I can't think of a good term for it.

Reply:
Been a while without looking into current's web code but shouldn't this be translated?

Reply:
How about renaming DirectStream to Audio Transcode, as that actually describes what happens?

Reply:
That's a fair compromise. Off the top of my head,  the ffmpeg logs on backend would need to renamed. I'm sure there is other places that'd need it too